builds/check_zproject/ci_build.sh

```diff
@@ -9,7 +9,7 @@ set -ex
 # Note: certain zproject scripts that deal with deeper dependencies expect that
 # such checkouts are directly in the same parent directory as "this" project.
 cd "$REPO_DIR/.."
-git clone --quiet --depth 1 https://github.com/42ity/libsodium.git libsodium
+git clone --quiet --depth 1 -b stable https://github.com/42ity/libsodium.git libsodium
```

Comment:
this is strange, if you have a look at PR #43, I already updated the project.xml in order to remove the default branch on all dependencies. It happened in many other repos that after deleting the default branch option it appeared back again, I will investigate more on this